src/Tools/Setup.php

```diff
@@ -77,6 +78,7 @@ public static function createAnnotationMetadataConfiguration(
     ) {
         $config = self::createConfiguration($isDevMode, $proxyDir, $cache);
         $config->setMetadataDriverImpl($config->newDefaultAnnotationDriver($paths));
+        AnnotationDriver::registerAnnotationClasses();
```

Comment:
I think it would be cleaner to use something like this (as per http://stackoverflow.com/a/9615926):

```php
$config = self::createConfiguration($isDevMode, $proxyDir, $cache);
$driver = $config->newDefaultAnnotationDriver($paths);
$config->setMetadataDriverImpl($driver);
$driver->registerAnnotationClasses();
```

This way you don't have to use the static accessor.